src/test/resources/features/Scoring/Report/Report.feature

```diff
@@ -0,0 +1,130 @@
+Feature: Report Feature
+
+  Background:
+    Given user prepare auth request
+    And user do login with email "admin@admin.com" and password "administratorfunctionapp"
+    And user prepare report request with batchCode "futur3"
```

Comment:
Also, does this background require batch with batchCode `futur3` to exist prior to running this feature's tests?

Reply:
yes